mitodl/micromasters
# Make OPEN_DISCUSSIONS_COOKIE_NAME required

#### What are the relevant tickets?
Related to https://github.com/mitodl/open-discussions/pull/285#discussion_r144308857

#### What's this PR do?
Makes `OPEN_DISCUSSIONS_COOKIE_NAME` required and removes its default value. On production environments cookies may be shared so we need to keep track that each cookie is different

#### How should this be manually tested?
Change the cookie name here and on open-discussions and verify that the login is successful.
